### PR TITLE
Enable traffic light detection using YOLO model

### PR DESCRIPTION
**Pull request by aozo**
### Summary of Changes:
* Enabled YOLO detector for traffic light detection
* Added new conf file for traffic light detection
* Modified control plot script to keep axis fixed

### CODE REVIEW

File: plot_control.py [hunk 1]
```diff
@@ -32,10 +32,10 @@ def main():
...
 
         # Config each subplot
-        df.plot(x='timestamp', y='speed', ax=axes[0][0], ylabel='km/h')
-        df.plot(x='timestamp', y='steer', ax=axes[0][1], ylabel='rad')
-        df.plot(x='timestamp', y='throttle', ax=axes[1][0])
-        df.plot(x='timestamp', y='brake', ax=axes[1][1])
+        df.plot(x='timestamp', y='speed', ax=axes[0][0], ylabel='km/h', ylim=(0, 90))
+        df.plot(x='timestamp', y='steer', ax=axes[0][1], ylabel='rad', ylim=(-0.5, 0.5))
+        df.plot(x='timestamp', y='throttle', ax=axes[1][0], ylim=(0, 1))
+        df.plot(x='timestamp', y='brake', ax=axes[1][1], ylim=(0, 1))
```
Comment (aozo) [inline plot_control.py:38]: Keep axis in control plot fixed.

File: pylot.py [hunk 2]
```diff
@@ -176,10 +176,21 @@ def driver():
...
+        # use detected obstacles such as speed signs to control the vehicle
+        waypoints_stream = pylot.component_creator.add_planning(
+            goal_location, pose_stream, prediction_stream, obstacles_stream,
+            lane_detection_stream, open_drive_stream, global_trajectory_stream,
+            time_to_decision_loop_stream)
+    else:
+        # Otherwise, keep the default behaviour which is the planning operator
+        # uses detected traffic lights to control the vehicle
+        waypoints_stream = pylot.component_creator.add_planning(
+            goal_location, pose_stream, prediction_stream, traffic_lights_stream,
```
Comment (aozo) [inline pylot.py:191]: Depending on if obstacle detection is enabled or not, vehicle planning will use either detected obstacles (speed signs) from the obstacles stream or traffic lights from the traffic light stream. Without this change, the vehicle will not react to traffic lights even if traffic light detection is enabled via the Pylot flags.

More work is required to enable simultaneous detection of speed signs and traffic lights and having the vehicle react to everything.

File: plot_control.py [hunk 3]
```diff
@@ -32,10 +32,10 @@ def main():
...
 
         # Config each subplot
-        df.plot(x='timestamp', y='speed', ax=axes[0][0], ylabel='km/h')
-        df.plot(x='timestamp', y='steer', ax=axes[0][1], ylabel='rad')
-        df.plot(x='timestamp', y='throttle', ax=axes[1][0])
-        df.plot(x='timestamp', y='brake', ax=axes[1][1])
+        df.plot(x='timestamp', y='speed', ax=axes[0][0], ylabel='km/h', ylim=(0, 90))
+        df.plot(x='timestamp', y='steer', ax=axes[0][1], ylabel='rad', ylim=(-0.5, 0.5))
+        df.plot(x='timestamp', y='throttle', ax=axes[1][0], ylim=(-0.1, 1.1))
+        df.plot(x='timestamp', y='brake', ax=axes[1][1], ylim=(-0.1, 1.1))
```
Comment (aozo) [inline plot_control.py:38]: Change to fix axis of control plots.

**Comment by abrahamchan** [comment]
LGTM!

**Pull request merged**